osnma-longan-nano/README.md

```diff
@@ -81,7 +81,14 @@ The serial port client will send the INAV and OSNMA data to the board and print
 to the standard output all the lines received from the microcontroller through
 the UART. The UART communication is described below.
 
-After starting the serial port client, the microcontroller should be resetted or
+So, in the same way that navigation data can be piped to this application running on a local computer ([instructions](https://github.com/daniestevez/galileo-osnma#quick-start-using-galmon)), a web stream or 'live' data from a GNSS receiver can be piped to the [Longan nano](https://longan.sipeed.com/en/) via the [serial port client](https://github.com/daniestevez/galileo-osnma/tree/main/osnma-longan-nano-client).
+
+Example: (run from ~/galileo-osnma/osnma-longan-nano-client/target/release/)
+```
+nc 86.82.68.237 10000 | ./osnma-longan-nano-client /dev/ttyUSB0
+```
```

Comment:
Here maybe we want a note saying that the computer's serial port may also be `/dev/ttyACM0` or something else (it deends on what UART to USB adapter you have).